src/wallet/mod.rs

```diff
@@ -438,11 +438,21 @@ where
         self.database.borrow().get_tx(txid, include_raw)
     }
 
-    /// Return the list of transactions made and received by the wallet
+    /// Return an unsorted list of transactions made and received by the wallet
     ///
     /// Optionally fill the [`TransactionDetails::transaction`] field with the raw transaction if
     /// `include_raw` is `true`.
     ///
+    /// To sort transactions, the following code can be used:
+    /// ```no_run
```

Comment:
Even if it's no_run it's still compiled (and ATM it fails), but you can make it work adding:

```
# let mut tx_list: Vec<bdk::TransactionDetails> = vec![];
```

This way, tx_list is declared, but it's not shown in the docs

Reply:
Done in https://github.com/bitcoindevkit/bdk/pull/686/commits/192f1cd1bf00f66a8bea5484b7d6b00a1b64c7da. Thanks.